workers/pyproject.toml

```diff
@@ -30,6 +30,8 @@ httpx = "^0.17.0"
 asyncpg = "^0.22.0"
 vcrpy = "^4.1.1"
 python-dateutil = "^2.8.2"
+scipy = "^1.7.1"
```

Comment:
Is there a reason you don't use the most recent version?

Reply:
Not sure why this came about. I will update to the most recent version.